src/query/storages/system/src/columns_table.rs

```diff
@@ -228,7 +272,10 @@ impl ColumnsTable {
                     _ => {
                         let schema = table.schema();
                         let field_comments = table.field_comments();
+                        let columns_statistics =
```

Comment:
Catalog will call disable table info refresh in this file. So the stat is correct? @dantengsky